docs/pages/developer-guide.md

```diff
@@ -939,6 +939,37 @@ In the example above, a server task can be accessed through the address:
 server-lb.hello-world.l4lb.thisdcos.directory:80
 ```
 
+
+## Virtual networks
+
+The SDK allows pods to join virtual networks, with the `dcos` virtual network available by defualt. You can specify that a pod should join the virtual network by adding the following to your service spec YAML:
+
+```yaml
+pods:
+  pod-on-virtual-network:
+    count: {{COUNT}}
+    # join the 'dcos' virtual network
+    networks:
+      dcos:
```

Comment:
Could we also add the `labels` here e.g.: 
```
    networks:
      dcos:
       labels: "key0:val0"
```

Then we mention that the labels are passed to the CNI plugins and interpreted.